stock-app/src/main/java/edu/JasonFlinkstrom/advancedjava/stock/StockApp.java

```diff
@@ -0,0 +1,24 @@
+package edu.JasonFlinkstrom.advancedjava.stock;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+* StockApp class implements StockService interface
+* NOTE: does not work yet

```

Comment:
don't submit 'done' code that doesn't work yet. 
perhaps you mean, that this class does what it needed to do for this assignment but that's not much?